src/components/Profile/ProfileInfo/ProfileInfo.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+
+import BACKGROUND_IMG from '../../../url_utils';
+
+import style from './ProfileInfo.module.css';
+
+
```

Comment:
Redundant empty line here - which means you haven't used eslint script from package.json.

Reply:
I tried using the airbnb style code settings. Look at it. It's ok?